docs/platforms/javascript/guides/nuxt/features/vue-router.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Vue Router
+description: "Learn about Sentry's Vue Routing integration."
+---
+
+Nuxt uses the vue-router by default, and our Nuxt SDK instruments your routing automaitcally, to parameterize transaction according to your routes.
+
+Our routing instrumentation supports `vue-router` v2, v3, and v4.
+
+<SignInNote />
+
+
+## Configuration
+
+You can pass an optional configuration object as a second argument to the browser tracing integration:
+
+```javascript
+Sentry.browserTracingIntegration({
+  router,
+  routeLabel: "path",
+});
+```
+
+The available options are:
+
+| Key        | Type   | Default | Description                                                                                                                                                                                                                                                           |
+| ---------- | ------ | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| routeLabel | string | `name`  | The label to use for the route transactions. Can be either `name` or `path`. When this is `name`, the transaction will use `route.name`, if it is set, and else use the path of the route. By setting this to `path` you can opt-out of this and always use the path. |
```

Comment:
M: We should delete this part because there are no custom options users can pass for the router in Nuxt. This all happens inside the SDK.

Reply:
done